apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,22 +1,13 @@
 <div
   class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
   [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+  <ng-content select="img"></ng-content>
 
   <section>
-    <app-list-item
+    <ng-container
       *ngFor="let item of list"
```

Comment:
you can now use the new control flow 😍

Reply:
you refer to @if, @for right?

Reply:
yes

Reply:
But is this content projection, with the new feats of Angular still being used? or have new solutions been proposed? simpler